src/backend/database_migrations/versions/20210727_111359_add_name_field_to_phylo_run_table.py

```diff
@@ -0,0 +1,26 @@
+"""Add name field to phylo run table
+
+Create Date: 2021-07-27 11:13:59.925534
+
+"""
+import enumtables  # noqa: F401
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "20210727_111359"
+down_revision = "20210617_201052"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        "phylo_runs",
```

Comment:
does the name go on the run, or on the tree?

Reply:
yeah i think it was the phylo_tree name